app/models/ability/membership.rb

```diff
@@ -21,9 +21,11 @@ def initialize(user)
     end
 
     can [:remove_admin, :revoke, :destroy], ::Membership do |membership|
-      (membership.user == user ||
-       user_is_admin_of?(membership.group_id) ||
-       (membership.inviter == user && !membership.accepted_at?))
+      user.is_admin || (
```

Comment:
I wonder if this should also check that the user is a bot. I'll have a think.. 

The reason i say this is that, generally we don't want site admins getting different behaviors than other users, except from within the admin panel.

Reply:
Right, constraining it only for bots would work for us too so if it makes more sense to you I can change in both places. Although I would say site admins can even impersonate other users so not sure if it would make a lot of difference.
Another thing to consider is that people can mark themselves as "bot" in the profile since it's been used to temporarily disable them from being added to polls too, which would give them this permission if they are admin as a side effect. Maybe we need another check to differentiate bot and who is on leave.

Reply:
Thanks for waiting for me to think this though. You're right,iIt's pointless if you an just change the value.